bedrock/externalpages/pocket_urls.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from bedrock.mozorg.util import page
+
+urlpatterns = (
+    page("about/", "externalpages/pocket/about.html"),
+    page("add/", "externalpages/pocket/add.html"),
+    page("android/", "externalpages/pocket/android.html"),
+    page("ios/", "externalpages/pocket/ios.html"),
+    page("chrome/", "externalpages/pocket/chrome.html"),
+    page("safari/", "externalpages/pocket/safari.html"),
+    page("opera/", "externalpages/pocket/opera.html"),
+    page("edge/", "externalpages/pocket/edge.html"),
+    page("welcome/", "externalpages/pocket/welcome.html"),
+    page("contact-info/", "externalpages/pocket/contact-info.html"),
+    page("firefox/new_tab_learn_more/", "externalpages/pocket/firefox/new-tab-learn-more.html"),
+    page("pocket-and-firefox/", "externalpages/pocket/pocket-and-firefox.html"),
+    page("get-inspired/", "externalpages/pocket/get-inspired.html"),
+    page("jobs/", "externalpages/pocket/jobs.html"),
+    page("privacy/", "externalpages/pocket/privacy.html"),
+    page("tos/", "externalpages/pocket/tos.html"),
+    page("save-to-pocket/", "externalpages/pocket/save-to-pocket.html"),
+)
```

Comment:
Moving the pocket URLs to being their own module, rather than in `externalpages.urls`, means we can plug it in to the default urlconf AND also use it in focused pocket-only urls.py - see later on

Reply:
Maybe we can just change `externalpages` to `pocket` to make things more clear

Reply:
- [ ] @stevejalim move pocket pages into their own higher-level folder

@pmac To keep it consistent I may also move views, not just the templates. Y'dig?

Reply:
I would move everything and just have a `pocket` app, yeah. I'm suggesting we rename `externalpages` app to `pocket` and make it just everything `pocket` for easy finding and reasoning about.

Reply:
I think it probably makes sense to move the GA testing template to the `mozorg` app and make sure it's noindex still

Reply:
Oh yeah. I agree. Testing templates don't belong here.

Reply:
Have asked Nathan, on his PR, to move his testing URL